css/at-rules/page.json

```diff
@@ -83,14 +83,16 @@
           "__compat": {
             "description": "<code>page-orientation</code> descriptor",
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page/page-orientation",
+            "spec_url": "https://w3c.github.io/csswg-drafts/css-page/#page-orientation-prop",
             "support": {
               "chrome": {
                 "version_added": "85"
               },
               "chrome_android": "mirror",
               "edge": "mirror",
               "firefox": {
-                "version_added": false
+                "version_added": "preview",
+                "notes": "<code>page-orientation</code> released in Firefox Nightly 111."
```

Comment:
I don't think we need this note, since we already set `preview`.

Reply:
Done